internal/testutils/helpers.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testutils
+
+// ToPtr returns a pointer to the given value of any type.
+func ToPtr[T any](v T) *T {
+	return &v
+}
+
+// StringPtr is a convenience wrapper around Ptr for string values.
+func StringPtr(v string) *string {
```

Comment:
Is this really needed?
You can do the same with `ToPtr()` directly and its clear enought imo.